### DISCUSSION TITLE
How to get a keystroke?

### DISCUSSION THREAD

**Discussion by Kollo4455** [ixjf/MSIRGB#241] [General]
I want to make a color switch when clicking on the left mouse button , how can this be implemented ?

**Comment by ixjf** [comment]
This can't be done directly in Lua. You'd need to write a C module to plug into Lua. It's also not so easy to do that. You'd need to either check very often for mouse button presses (often enough that you wouldn't miss a button press), or, alternatively, hook into the system-wide window message for mouse clicks. There doesn't seem to be any Lua modules already written for that.
